tests/test_actions.py

```diff
@@ -811,23 +812,6 @@ def test_create__conflict(self, read_order):
 
         assert "In Level1" in e.value.get_display_string()
 
-    @pytest.mark.parametrize("read_order", ATTR_CONFLICT_ORDERING_CASES)
```

Comment:
Instead of removing this test, please fix it.  The way this works is there are two "SELECT_OBJECT_1" occurring in different orders in the cases ATTR_CONFLICT_ORDERING_CASES.  Please duplicate "SELECT_OBJECT_1" as "SELECT_OBJECT_1_CONFLICT" and change the field value.